platformio.ini

```diff
@@ -19,31 +19,30 @@ monitor_speed = 115200
 ; common environments
 ;--------------------
 
+[env_common]
```

Comment:
this and most changes below in the platformio.ini seem to be left over from legacy branch version
I think the disection into rx and tx as in main should remain

Reply:
I didn't see any Tx references in main but now the Rx stuff is there.

Reply:
correct, no tx there, but the separation to make that easily possible